### ISSUE TITLE
Methane cross section

### ISSUE THREAD

**Issue by hgamarro** [hgamarro/WRF_Landfill#9]
Write python code to generate any cross section based on lat lon.

**Comment by hgamarro** [comment]
Use this link as template 
https://wrf-python.readthedocs.io/en/latest/plot.html#vertical-cross-section

and load in data from Parker. @parkerY99 can you give us the path to the most up to date wrfouts with emission working?

**Comment by parkerY99** [comment]
test1

**Comment by melius30** [comment]
added first nb 0a2fb67f4a5c0b28ac1b6cb5ec27a234aa37390b

**Comment by hgamarro** [comment]
Figure out why we have a gap in the vertical cross section, use one of Parkers newest wrfout that has 90 vertical levels

**Comment by parkerY99** [comment]
/global/scratch/users/kwyang0331/Chemtracer/afterOSupdate/WRFchem/test/em_real/results/1031tker

use the following Directory to use WRFout files which have Vert_level: 100

**Comment by melius30** [comment]
I updated the vertical cross-section script. I figured out that the vertcross function, by default, uses 99 vertical levels , which are relatively uniform (e.g., 0, 203, 406, 609, ..., 20000). That's why, the vertical cross-section starts at 203 m and has large gaps between levels. To address this, I modified the script to include the level input directly within the vertcross function, matching the levels of the z (height) variable (e.g., 0, 30, 87, 150, ..., 20000), which are much denser near the surface. The vertical cross section looks fine now
